js/views/userPageVw.js

```diff
@@ -812,6 +839,17 @@ module.exports = baseVw.extend({
     }
   },
 
+  renderReviews: function (model) {
+    model = model || [];
+    this.reviewsVw = new reviewsView({
```

Comment:
Since this may be called multiple times, please be sure to remove any existing reviewsVw before creating a new one:

```
this.reviewsVw && this.reviewsVw.remove()
```